kronjob/_kronjob.py

```diff
@@ -102,6 +102,7 @@ def serialize_k8s(k8s_object):
 
 
 def build_k8s_object(aggregate_job):
+    _ALTERNATE_DEFAULTS['containerName'] = '{}-job'.format(aggregate_job['name'])
```

Comment:
should check to make sure this is under 63 chars in length, think kubernetes has a limit on container names, and there are other issues (see https://github.com/better/mortgage/pull/18124)

probably should make the max length configurable